exp-player/addon/components/exp-overview/component.js

```diff
@@ -107,13 +107,16 @@ export default ExpFrameBaseComponent.extend(Validations, {
     type: 'exp-overview',
     layout: layout,
     questions: questions,
-    responses: Ember.computed('questions', function() {
-        var questions = this.get('questions');
-        var responses = {};
-        for (var i=0; i < questions.length; i++) {
-            responses[i] = questions[i].value;
-        }
-        return responses;
+    showOptional: Ember.computed('questions.7.value', function() {
+       return this.questions[7].value === 'global.yesLabel';
+    }),
+    responses: Ember.computed(function() {
```

Comment:
Watching `questions` would just watch for changes to the list itself, instead of changes to the `value` of each question. It looks like `responses` gets re-computed when you click `continue` so all of the answers are there.